pkg/testing/integration_register.go

```diff
@@ -70,6 +70,21 @@ var _ = Suite(&MySQL{
 	},
 })
 
+// cockroachdb app
+type CockroachDB struct {
+	IntegrationSuite
+}
+
+var _ = Suite(&CockroachDB{
+	IntegrationSuite{
+		name:      "cockroachdb",
+		namespace: "cockroachdb-test",
+		app:       app.NewCockroachDB("cockroachdb"),
+		bp:        app.NewBlueprint("cockroachdb", "", false),
```

Comment:
Are we not going to use the dev image for kanister-tools in the blueprint during test? I think we should do that.

Reply:
we're using cockroachdb/cockroach image in KubeTask for Backup and Restore, so if it's set to true, it's changing the tag from required v22.1.5 to v9.99.9-dev, and there's no such tag compatible for image cockroachdb/cockroach